docs/spark-writes.md

```diff
@@ -34,16 +34,16 @@ Some plans are only available when using [Iceberg SQL extensions](../spark-confi
 
 Iceberg uses Apache Spark's DataSourceV2 API for data source and catalog implementations. Spark DSv2 is an evolving API with different levels of support in Spark versions:
 
-| Feature support                                  | Spark 3 | Notes                                        |
-|--------------------------------------------------|-----------|----------------------------------------------|
-| [SQL insert into](#insert-into)                  | ✔️        |                                              |
-| [SQL merge into](#merge-into)                    | ✔️        | ⚠ Requires Iceberg Spark extensions          |
-| [SQL insert overwrite](#insert-overwrite)        | ✔️        |                                              |
-| [SQL delete from](#delete-from)                  | ✔️        | ⚠ Row-level delete requires Spark extensions |
-| [SQL update](#update)                            | ✔️        | ⚠ Requires Iceberg Spark extensions          |
-| [DataFrame append](#appending-data)              | ✔️        |                                              |
-| [DataFrame overwrite](#overwriting-data)         | ✔️        |                                              |
-| [DataFrame CTAS and RTAS](#creating-tables)      | ✔️        |                                              |
+| Feature support                                  | Spark 3 | Notes                                             |
+|--------------------------------------------------|-----------|---------------------------------------------------|
+| [SQL insert into](#insert-into)                  | ✔️        | ⚠ Requires `spark.sql.storeAssignmentPolicy=ANSI` |
+| [SQL merge into](#merge-into)                    | ✔️        | ⚠ Requires Iceberg Spark extensions               |
+| [SQL insert overwrite](#insert-overwrite)        | ✔️        | ⚠ Requires `spark.sql.storeAssignmentPolicy=ANSI` |
+| [SQL delete from](#delete-from)                  | ✔️        | ⚠ Row-level delete requires Spark extensions      |
+| [SQL update](#update)                            | ✔️        | ⚠ Requires Iceberg Spark extensions               |
+| [DataFrame append](#appending-data)              | ✔️        |                                                   |
+| [DataFrame overwrite](#overwriting-data)         | ✔️        |                                                   |
+| [DataFrame CTAS and RTAS](#creating-tables)      | ✔️        | ⚠ Requires v2 tables                              |                                      
```

Comment:
```suggestion
| [DataFrame CTAS and RTAS](#creating-tables)      | ✔️        | ⚠ Requires v2 tables                              |
```

Reply:
we should maybe link to how to create a V2 table as it isn't obvious for new users

Reply:
Sorry, I meant `DSv2 API`. Updated.